wiki/en/Include-Shared-Commands.md

```diff
@@ -2,11 +2,10 @@
 [comment]: # (This is an include file for use in multiple documents)
 
 - `-h` or `--help`     Display help text         
-- `-i` or `--inifile`   Set location of initialization file (overrides default) 
+- `-i` or `--inifile`  Set location of initialization file (overrides default) 
 - `-n` or `--nogui`    Disable GUI (for use in headless mode)                      
-- `-p` or `--port`    Sets the local UDP port number. Default is 22124 
-- `-Q` or `--qos`     Sets the quality of service DSCP value. Default is 128. Disable with 0. QoS is ignored by Windows. To enable it, see
-section below
+- `-p` or `--port`     Sets the local UDP port number. Default is 22124 
+- `-Q` or `--qos`      Sets the quality of service DS Field byte. Default is 128 (DSCP/CS4). QoS is ignored by Windows. To enable it, see section below.
 - `-t` or `--notranslation`   Disable UI language translations
--  `-6` or `--enableipv6`      Enable IPv6 addressing (IPv4 is always enabled)
-- `-v` or `--version`   Output version information and exit 
+- `-6` or `--enableipv6`      Enable IPv6 addressing (IPv4 is always enabled)
+- `-v` or `--version`  Output version information and exit 
```

Comment:
Spacing seems a bit odd here.

Reply:
It looks odd, but it is OK. All that I did was remove a space in each line as seen by the dark pink squares.
Here is what the file looks like:

wiki/en/Include-Shared-Commands.md
```

[comment]: # (This is an include file for use in multiple documents)

- `-h` or `--help`     Display help text         
- `-i` or `--inifile`  Set location of initialization file (overrides default) 
- `-n` or `--nogui`    Disable GUI (for use in headless mode)                      
- `-p` or `--port`     Sets the local UDP port number. Default is 22124 
- `-Q` or `--qos`      Sets the quality of service DS Field byte. Default is 128 (DSCP/CS4). QoS is ignored by Windows. To enable it, see section below.
- `-t` or `--notranslation`   Disable UI language translations
- `-6` or `--enableipv6`      Enable IPv6 addressing (IPv4 is always enabled)
- `-v` or `--version`  Output version information and exit 
```

Reply:
Ok. It’s still a bit strange but seems to work. Maybe we should get rid of the spacing all together (but that’s up to another PR and decision)